src/data/javascript-quiz.ts

```diff
@@ -1968,6 +1968,18 @@ const javascriptQuiz = [
     Explanation:
       "An IIFE or Immediately Invoked Function Expression, is used to declare a private variable and to avoid polluting the global environment. If a function is declared with an IIFE by using a parenthesis around it, the variable declared inside it becomes private and its value can be accessed using a getter function inside the original function.",
     Link: "https://developer.mozilla.org/en-US/docs/Glossary/IIFE"
+  },
+  {
+    Question:
+      "In JavaScript, which of the following statements is correct regarding async await?",
+    Answer: "async automatically returns a promise",
+    Distractor1: "await keeps executing code regardless of promies status",
```

Comment:
found a spelling error

```suggestion
    Distractor1: "await keeps executing code regardless of promise status",
```